seed/challenges/01-front-end-development-certification/html5-and-css.json

```diff
@@ -1041,7 +1042,7 @@
       ],
       "tests": [
         "assert($(\"h2\").css(\"font-family\").match(/^\"?lobster/i), 'message: Your h2 element should use the font <code>Lobster</code>.');",
-        "assert($(\"h2\").css(\"font-family\").match(/lobster.*,.*monospace/i), 'message: Your h2 element should degrade to the font <code>Monospace</code> when <code>Lobster</code> is not available.');",
+        "assert(/\\s*?h2\\s*?\\{\\s*?font-family\\:\\s*?(\\'|\\\")?Lobster(\\'|\\\")?,\\s*?Monospace\\s*?\\;\\s*?\\}/gi.test(code), 'message: Your h2 element should degrade to the font <code>Monospace</code> when <code>Lobster</code> is not available.');",
```

Comment:
Oops, yes, I do! Thank you. Sorry for the mess up there. 😅 
I edited my original message now.
_Sidenote: this looked really funny in the email. It showed "do you mean \\s*? === \\s*?"_